claude_api/client.py

```diff
@@ -154,23 +160,26 @@ def __init__(
         self.timezone: str = get_localzone().key
 
     def __get_proxy(self) -> dict[str, str] | None:
-
-        if self.proxy is None or not issubclass(self.proxy.__class__, ClaudeProxy):
+        if self.proxy is None or not isinstance(self.proxy, ClaudeProxy):
             return None
```

Comment:
Would like to make sure the instance of the proxy is either HTTPPRoxy or SOCSProxy